### DISCUSSION TITLE
Possible reasons for ephemeral pop-up windows not closing.

### DISCUSSION THREAD

**Discussion by jeffmax** [Xpra-org/xpra#4295] [Q&A]
We are testing out 3D Slicer (https://www.slicer.org/) using Xpra 5.0.9 and the bundled HTML5 client on Jammy (and Chrome for the browser). It is working well, but there is a situation where the application will open a pop-up window briefly that is meant to show progress on the import of a file. On other platforms, this pop up window opens quickly, displays a progress bar and closes automatically. In Xpra (in seamless mode), the pop-up stays. It is still responsive and can be moved and manually closed, but it does not close automatically. I was wondering if this behavior hints at a root cause? Perhaps the application is depending on a feature of a window manager that is not present? Any suggestions would be helpful. Thanks!

**Comment by totaam** [comment]
Is this only with the html5 client or also with the Python client?

**Comment by jeffmax** [comment]
Just tested this with the Python client and the behavior was the same. The window in the image with title "Processing..." persists when executed through Xpra but closes automatically when executed natively.

![image](https://github.com/Xpra-org/xpra/assets/479260/02a0f22f-1f90-44db-98a7-5d5b51d204f3)

**Comment by totaam** [comment]
Sorry, no idea.
